01-JS-DOM-APIs/exerciseJS.js

```diff
@@ -0,0 +1,75 @@
+//EXERCISE 6
+  function eventRequest() {
+    var xhttp = new XMLHttpRequest();
+    xhttp.open('GET', 'http://api.icndb.com/jokes/random', true);
+    xhttp.onreadystatechange = function(event) {
+      var response = JSON.parse(event.target.response);
+      document.getElementById('sectionHidden').innerHTML = response.value.joke;
+       if (xhttp.readyState === XMLHttpRequest.DONE) {
+         if (xhttp.responseText.value) {
+             document.getElementById("buttonSection").innerHTML  = xhttp.responseText.value.joke;
+         }
+       }
+   }
+   xhttp.send();
+}
+//EXERCISE 7
+function eventOnClick(){
```

Comment:
eventOnClick doesn't make sense on what the method do. This could be handleJokeButtonClick